docs/conf/portland/2019/news/announcing-speakers.rst

```diff
@@ -0,0 +1,137 @@
+:template: {{year}}/generic.html
+
+.. post:: Feb 27, 2019
+   :tags: portland-2019, cfp, talks
+
+Speaker Announcement
+====================
+
+Happy Wednesday everybody!
+
+We’re less than three months away from Write the Docs Portland 2019
+conference! We have so much news to share, so without further adieu,
+here we go!
+
+First and foremost, **we’re very excited to announce our finalized speaker
+list for the conference!** Our speaker lineup this year is packed with a
+diverse and compelling melange of documentarians who have amazing
+stories to share with conference attendees. We feel that with our
+speakers – whose expertise spans across many fields and industries –
+will share interesting and actionable takeaways for attendees.
+
+We want to thank everyone who took time to send us proposals. Each year,
+talk selection gets increasingly difficult as we receive a larger,
+strong batch of submissions. We had many spirited discussions over weeks
+about the speaker lineup, and it is never easy to decide on final list
+of speakers. Remember that Write the Docs Portland would not be possible
+without your support and proposals. Please consider submitting your
+talks to Write the Docs meetup near you, as they are always looking for
+more speakers!
+
+With a great speaker lineup just released, now is the time to purchase
+your Write the Docs Portland 2019 conference ticket. Act soon, as
+tickets tend to sell out weeks before the conference! And while you’re
+at the conference, be sure to spend some time exploring the magical city
+of Portland.
+
+Finally, our generous sponsors make Write the Docs Portland 2019
+conference possible. Interested in having your company as a conference
+sponsor? Read on!
+
+Full speaker line-up
+--------------------
+
+Every year, we look to bring a diversity of voices to the Write the Docs
+main stage. Because the role of "documentarian" is different for each of
+us, we strive to craft a lineup that spans a good mix of practical,
+philosophical, and technical topics.
+We're really excited about the presentations this year – we hope you are too!
+
+
```

Comment:
The links go to a "nothing to see yet" page (I presume as planned, until you get the descriptions up in a different PR?)

Reply:
It's part of this PR -- you can see it in the preview: https://deploy-preview-801--dentist-canary-64055.netlify.com/conf/portland/2019/speakers/